spec/openapi.yml

```diff
@@ -286,6 +310,39 @@ paths:
         500:
           $ref: "#/components/responses/500"
 
+  /sandboxes/{sandboxID}/logs:
+    get:
+      description: Get sandbox logs
+      tags: [sandboxes]
+      security:
+        - ApiKeyAuth: []
+      parameters:
+        - $ref: "#/components/parameters/sandboxID"
+        - in: query
```

Comment:
can we also add end? We may not use it right now, but it IMHO makes a lot of sense, especially if the sandbox ran for several hours, for sharing specific logs with colleagues etc.

Reply:
Right now we are not exposing the start in CLI. I thought about adding this, but let's not invest more work until necessary — we can add the flags and `end` parameter whenever we want without breaking anything.

Let's see how useful this is first.